app/src/main/java/jp/developer/retia/frozenword/ui/watch/WatchScreen.kt

```diff
@@ -45,32 +57,42 @@ fun PreviewHabbits() {
     }
 }
 
+@OptIn(ExperimentalMaterialApi::class)
 @Composable
 fun WatchScreen(
     modifier: Modifier = Modifier,
     title: String,
     simpleHabbitTitle: String,
     trigger: String? = null,
     place: String? = null,
+    logs: List<Log> = emptyList(),
     onCompleted: () -> Unit = {}
 ) {
-    Column(
-        modifier = modifier
-            .fillMaxWidth()
-            .fillMaxHeight(),
-        verticalArrangement = Arrangement.Center,
-        horizontalAlignment = Alignment.CenterHorizontally,
-    ) {
-        Text(title, style = MaterialTheme.typography.h6)
-        trigger?.let {
-            Text(trigger, style = MaterialTheme.typography.h6)
-        }
-        place?.let {
-            Text(place + "で", style = MaterialTheme.typography.h6)
-        }
-        Text(simpleHabbitTitle, style = MaterialTheme.typography.h5)
+    val scope = rememberCoroutineScope()
+    val scaffoldState = rememberBottomSheetScaffoldState()
+    BottomSheetScaffold(
+        scaffoldState = scaffoldState,
+        sheetContent = {
+            LogsScreen(logs = logs)
+        }) {
```

Comment:
```suggestion
        }
    ) {
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>